lib/html-import-template-bundler.js

```diff
@@ -28,17 +28,19 @@ export function bundle(cfg) {
       cwd: baseURL.replace(/\\/g, '/')
     })
     .forEach(function(file) {
-      file = path.resolve(baseURL, file);
-      var content = fs.readFileSync(file, {
-        encoding: 'utf8'
-      });
-
-      var $ = whacko.load(content);
-      var name = getCanonicalName(builder, file, 'view').replace(/!view$/g, '');
-
-      $('template').attr('id', name);
-      var template = $.html('template');
-      templates.push(template);
+      if(file != '.') {
```

Comment:
Here is my bundle configuration.
```
{
  "bundles": {
    "dist/app-build": {
      "includes": [
        "[**/*]"
      ],
      "options": {
        "inject": true,
        "minify": true
      }
    },
	"dist/app-bundle": {
		"htmlimport": true,
		"includes": ["dist/**/*.html"],
		"options": {
			"inject": true
		}
	},
    "dist/aurelia": {
      "includes": [
        "aurelia-framework",
        "aurelia-bootstrapper",
        "aurelia-fetch-client",
        "aurelia-router",
        "aurelia-animator-css",
        "aurelia-templating-binding",
        "aurelia-templating-resources",
        "aurelia-templating-router",
        "aurelia-loader-default",
        "aurelia-history-browser",
        "aurelia-logging-console",
		"aurelia-http-client",
		"aurelia-html-import-template-loader",
		"aurelia-validation",
        "bootstrap",
        "bootstrap/css/bootstrap.css!text"
      ],
      "options": {
        "inject": true,
        "minify": false
      }
    }
  }
}
```

I've been able to reproduce this issue in 2 different projects with the same configurations on 4 Windows 10 desktops and 2 Windows Server 2012 build servers. I am using the html template loader for my html files and it seems to only be happening with the html portion of the bundle. After it goes through all of my html files it ends with the current directory ".". I'm using the following version:

`jspm 0.16.23`
`node 5.4.0`
`npm 3.3.12`

If I setup the npm to use a global version where I have modified the dist file to have this fix I no longer receive the error and the bundle completes with no issues.